ocaml/xapi/xapi_pool.ml

```diff
@@ -1620,6 +1620,20 @@ let eject ~__context ~host =
       (fun () -> Xapi_fuse.light_fuse_and_reboot_after_eject ()) ;
     Xapi_hooks.pool_eject_hook ~__context
 
+(** eject [host] from the pool. This code is run on all hosts in the
+pool but only [host] will eject itself. *)
+let eject ~__context ~host =
+  let local = Helpers.get_localhost ~__context in
+  let master = Helpers.get_master ~__context in
+  let host_uuid = Db.Host.get_uuid __context host in
+  match (host = local, host = master) with
+  | true, false ->
+      eject ~__context ~host
+  | false, _ ->
+      Certificates_sync.eject_certs_from_fs_for ~host_uuid
```

Comment:
Although it makes sense to call directly this function for all the `other` hosts in message_forwarding I'd rather keep it this way because it's not exposed in the API.